src/sidebar/components/ShareDialog/ImportAnnotations.tsx

```diff
@@ -156,28 +157,33 @@ function ImportAnnotations({
       <FileInput onFileSelected={setFile} disabled={busy} id={fileInputId} />
       {userList && (
         <>
-          <label htmlFor={userSelectId} className="block font-medium">
+          <p className="block font-medium">
             Select which user&apos;s annotations to import:
-          </label>
-          <Select
-            id={userSelectId}
-            data-testid="user-list"
-            disabled={busy}
-            onChange={e =>
-              setSelectedUser((e.target as HTMLSelectElement).value || null)
+          </p>
+          <SelectNext
+            value={selectedUser}
+            onChange={setSelectedUser}
+            label={
```

Comment:
I'm realizing that the prop name `label` is confusing here because we're actually using it display the _value_ of the selected item or a placeholder if no item is selected. This is distinct from "the control's label" (as in a `<label>`).

Reply:
Yeah, I've had that same thought a couple of times while using this.

On a simpler component you would use `children` for this, but since here `children` is the list of options, we need another prop for the actual displayed value.

Headlessui solves this problem by adding more sub-components. Specifically, it wraps all `Option` items in an `Options` wrapper, and then introduces a "dummy" `Button` component with the same purpose as our `SelectNext`'s `label` prop.

But I found this a bit over-complicated. I would suggest we stick with one prop in the upper-most component, but find a better name than `label`. Maybe `button` would follow a similar idea.

```tsx
<SelectNext value={value} onChange={...} button={<div>{value.whatever}</div>} />
```

Reply:
> But I found this a bit over-complicated. I would suggest we stick with one prop in the upper-most component, but find a better name than label. Maybe button would follow a similar idea.

Something like this works. Maybe `buttonContent` though since the component is not the button itself, but its contents/children.

Reply:
I like `buttonContent`. Tracked in https://github.com/hypothesis/frontend-shared/issues/1285